src/constant_texts.c

```diff
@@ -408,7 +408,7 @@ const char* ui_text_incompatible_card_version = "Incompatible card version! Cont
 const char* ui_text_card_error_contact_support = "Contact support & share the device logs";
 const char* ui_text_device_and_card_not_paired = "This device & card are not paired";
 const char* ui_text_security_conditions_not_met = "Security conditions not met!";
-const char* ui_text_process_reset_due_to_inactivity = "Process was reset due to inactivity";
+const char* ui_text_process_reset_due_to_inactivity = "Operation aborted due to inactivity";
```

Comment:
aborted does not sound good.

```suggestion
const char* ui_text_process_reset_due_to_inactivity = "Operation stopped due to inactivity";
```